cmd/cluster-kube-apiserver-operator/render/render.go

```diff
@@ -121,14 +125,27 @@ type TemplateData struct {
 
 	// EtcdServingCA is the serving CA used by the etcd servers.
 	EtcdServingCA string
+
+	// RestrictedCIDRs is a list of restricted CIDRs in the admissionPluginConfig
+	RestrictedCIDRs []string
 }
 
 // Run contains the logic of the render command.
 func (r *renderOpts) Run() error {
+	installConfigFileData, err := ioutil.ReadFile(r.configFile)
+	if err != nil {
+		return err
+	}
+	restrictedCIDRs, err := discoverRestrictedCIDRs(v1alpha1helpers.InstallConfigFromFile(installConfigFileData))
```

Comment:
would be simpler to just parse the file and then parse the ClusterConfig Go struct into discoverRestrictedCIDRs. Wdyt? Functional is nice, but does not really make it simpler here.

Reply:
I agree with removing the functionality, but because we still have to parse out the InstallConfig from the data in the config map I'd like to do that step as well and pass the InstallConfig struct directly instead (see changes to https://github.com/openshift/library-go/pull/77)